jibri/rootfs/defaults/jibri.conf

```diff
@@ -136,31 +150,4 @@ jibri {
       port = {{ $STATSD_PORT }}
     }
     {{ end -}}
-
```

Comment:
We didn't want these to disappear. This is https://github.com/jitsi/docker-jitsi-meet/pull/1170

Reply:
Re-added, tagged stable-8960-1-ems.2